From: Kornel <kornel@geekhood.net>
Date: Tue, 20 Mar 2018 13:56:32 +0000
Subject: Make static OpenMP linking opt-in

---
 Cargo.toml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 724595e..439f973 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -38,7 +38,8 @@ features = ["codec", "format", "filter", "software-resampling", "software-scalin
 
 [features]
 default-features = []
-openmp = ["imagequant/openmp-static"]
+openmp = ["imagequant/openmp"]
+openmp-static = ["imagequant/openmp-static"]
 video = ["ffmpeg"]
 
 [lib]
